microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -482,3 +482,9 @@ def _trigger_kdump_on_specified_cpu(
                 "The cpu count can't meet the test case's requirement. "
                 f"Expected more than {cpu_num} cpus, actual {cpu_count}"
             )
+
+    def after_case(self, log: Logger, **kwargs: Any) -> None:
```

Comment:
This can expand the argument like test case. `    def after_case(self, log: Logger, node: Node) -> None:`